composer.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "jetrails/magento-twofactor",
+    "description": "User based 2FA enablement for admin users in Magento 1.x",
+    "type": "magento-module",
+    "version": "1.1.4",
+    "license": "MIT",
+    "homepage": "https://github.com/jetrails/magento-twofactor",
+	"authors": [
+		{
+			"name": "Rafael Grigorian",
+			"email": "me@raffi.io"
+		}
+	],
+    "require": {
+        "magento-hackathon/magento-composer-installer": "*"
```

```suggestion
        "Cotya/magento-composer-installer": "*"
```

https://github.com/magento-hackathon/magento-composer-installer states that the repository is deprecated and points to https://github.com/Cotya/magento-composer-installer instead.

---

thanks @null93 

It's my understanding that while the namespace of the repo for `magento-composer-installer` has changed the namespace for the composer package hasn't
https://packagist.org/packages/magento-hackathon/magento-composer-installer
https://github.com/Cotya/magento-composer-installer/blob/main/composer.json#L2

https://packagist.org/search/?q=Cotya/magento-composer-installer

the package is still under `magento-hackathon/magento-composer-installer` (at least in packagist.org) so changing the namespace to `Cotya` would break it.

---

Say no more! I definitely missed that; thanks for taking the time to link references, and thanks again for the PR!